app/views/layouts/application.html.erb

```diff
@@ -10,10 +10,12 @@
 
   <body>
     <nav class = "topnav">
+      <%= link_to 'Return Home', '/' %>
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
+      <%= link_to 'Register', '/register/new' %>
+      <%= link_to 'Log In', '/login' %>
       <%= link_to "Cart: #{cart.total_items}", "/cart" %>
-      <%= link_to 'Register User', '/register/new' %>
```

Comment:
Let's create a break between these when we refactor.

Reply:
Agreed, good idea Hanna!